change/@ni-nimble-components-456f6771-229b-4052-ada2-7f642545afdf.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add fix to retain the format button state when user configuration gets dynamically updated",
```

Reminder that these comments appear in a changelog for the entire package so it's good to include the component name to help people understand the change scope.
```suggestion
  "comment": "Add fix to retain the rich text editor format button state when user configuration gets dynamically updated",
```

---

Updated